docs/pages/includes/config-reference/instance-wide.yaml

```
@@ -55,10 +55,18 @@ teleport:
         # token_name: /var/lib/teleport/tokenjoin
         token_name: "token-name"
 
-    # Optional CA pin of the auth server. This enables a more secure way of
-    # adding new nodes to a cluster. See "Adding Nodes to your Cluster"
-    # (https://goteleport.com/docs/setup/admin/adding-nodes).
-    ca_pin: "sha256:7e12c17c20d9cb504bbcb3f0236be3f446861f1396dcbb44425fe28ec1c108f1"
+    # Optional CA pin of the Auth Service, which enables new agents to trust a
```

```suggestion
    # Optional CA pin of the Auth Service. Specifying a CA pin enables new agents to trust a
```
This seems kind of important to place in a subordinate clause. Is